### PR TITLE
Adds wooden buckler shield

### PR DESCRIPTION
**Pull request by Bawhoppen**
I beat you to it anturk I'm sorry for getting your hopes up

Can be made from 10 wood functions like a riot shield 

:cl: 
add: Amateur medieval enthusiasts have found a quick way to easily make wooden bucklers.
/:cl:

lets hope I did my changelogs correctly this time

### CODE REVIEW

**Comment by Bawhoppen** [comment]
oh I cucked it give me one second

**Comment by Iamgoofball** [comment]
needs wooden sword

**Comment by Bawhoppen** [comment]
wait no I didn't cuck it phew

**Comment by Bawhoppen** [comment]
oh I should have made this burnable whatever fuck it

File: code/game/objects/items/weapons/shields.dm [hunk 1]
```diff
@@ -39,6 +39,16 @@
...
 	item_state = "roman_shield"
 
+/obj/item/weapon/shield/riot/buckler
+	name = "wooden buckler"
+	desc = "A medieval wooden buckler."
+	icon_state = "buckler"
+	item_state = "buckler"
+	materials = list()
+	slot_flags = null
+	force = 15
```
Comment (Iamgoofball) [inline code/game/objects/items/weapons/shields.dm]: >15 force

ahem

**Comment by GunHog** [comment]
Needs sprite

**Comment by Bawhoppen** [comment]
@GunHog there were already sprites in code so thats what inspired me

File: code/game/objects/items/weapons/shields.dm (hunk 1 above)
Comment (Bawhoppen) [inline code/game/objects/items/weapons/shields.dm]: I was going to say a meme but I forgot the exact words but 15 force isn't that much for bashing heads in

Comment (Bawhoppen) [inline code/game/objects/items/weapons/shields.dm]: oh here I found it https://www.youtube.com/watch?v=8OydCKdKlbM

Comment (Bawhoppen) [inline code/game/objects/items/weapons/shields.dm]: but i'll decrease it if its too much

Comment (Bawhoppen) [inline code/game/objects/items/weapons/shields.dm]: your silence is maddening going to fucking kill self and change it now

**Comment by Zilenan91** [comment]
15 is... a bit high but probably fine.

**Comment by Bawhoppen** [comment]
well I just changed it to be 10 and made riot shields 10 too so its probably fine

**Comment by Zilenan91** [comment]
riot shields did 15 before? Jeez, yeah that was way too high if that was the case.

**Comment by Bawhoppen** [comment]
no they did 8 but now they're 10 and so is the buckler

**Comment by KorPhaeron** [comment]
>rewrite blocking system so shields can have different values, block different kinds of attacks, react on hit to different attacks
>make the wood shield have the exact stats as the normal shield 

Please

**Comment by PKPenguin321** [comment]
anybody with a crowbar and library access can get these roundstart lol
free riotshields boys

**Comment by KorPhaeron** [comment]
The librarian buff we needed

**Comment by Zilenan91** [comment]
You could already get riot shields if you popped a coin in the costume vendor in the theatre

**Comment by PKPenguin321** [comment]
the difference is that's only one shield

**Comment by Dorsisdwarf** [comment]
Make this not work on ranged attacks at all (or like, 10%)

**Comment by Bawhoppen** [comment]
shields aren't that strong considering the stun combat we have

you can only get about 3 shields from library

**Comment by Bawhoppen** [comment]
I guess I'll increase the price and maybe make it have less blocking chance

**Comment by Zilenan91** [comment]
What's it at now?

**Comment by Bawhoppen** [comment]
force is 10 (same as toolbox, knife, etc), blocking chance is 40 (riot shield is 50 by comparison), 20 wood planks to build

**Comment by LatD** [comment]
One possible nerf could be adding a possibility for the wooden shield to get broken, if that's doable with the current shield code system

**Comment by Bawhoppen** [comment]
its probably possible but I don't know how to do it
if I did I'd let them be able to be set on fire while you're holding it

**Comment by KorPhaeron** [comment]
>if that's doable with the current shield code system

Yep. I made it all very flexible in the rewrite.

**Comment by LanCartwright** [comment]
Use changeling shield code for shield breaking.

**Comment by Bawhoppen** [comment]
I don't want to do shield breaking because nothing else that should break actually does in the game

**Comment by Bawhoppen** [comment]
it now can be worn on your back 
also christ git is painful

**Comment by tkdrg** [comment]
Sure why not let's give the librarian riot shields

**Pull request merged**